packages/types/src/interfaces/mmr/runtime.ts

```diff
@@ -64,38 +58,6 @@ export const runtime: DefinitionsCall = {
           ],
           type: 'Result<(), MmrError>'
         },
-        verify_proof: {
```

Comment:
The previous versions should not be removed. They can be available on older nodes.

The API decorates based on the `rpc_methods` call and will inject what is available on the RPC node.

Reply:
Taking account of your comment over here too:
https://github.com/polkadot-js/api/issues/5484#issuecomment-1430851882

The breaking change is deliberate since we've also reset MMR storage here: https://github.com/paritytech/devops/issues/2287
Thus, any proofs generated or supported by the old API will not bear meaning for the reset MMR.
I fully agree we should version the MMR RPC methods henceforth, but given the MMR reset, I reckon it's superfluous for this first iteration.

Reply:
The issue is that there are existing chains that could uses these RPCs - we cannot just remove what is there since it is not in master.

Old RPCs should stay since they are decorated on un-updated chains.